python/llm/dev/benchmark/all-in-one/run.py

```diff
@@ -159,12 +163,13 @@ def run_transformer_int4(repo_id,
             input_ids = tokenizer.encode(input_str, return_tensors="pt")
             input_ids = input_ids[:, :in_len]
             true_str = tokenizer.batch_decode(input_ids)[0]
-            input_ids = tokenizer.encode(true_str, return_tensors="pt")
+            input_ids = tokenizer.encode(true_str, return_tensors="pt")[:, :in_len]
```

Comment:
`true_str` is processed to match the input length, do we need [:, :in_len] here?

Reply:
Yes, in my local test, when without `[:, :in_len]`, the input length maybe 1 token longer than we need.
For example, for falcon, 1024 input actually is tested with 1025 length input id.